modules/installation-vsphere-infrastructure.adoc

```diff
@@ -44,7 +44,7 @@ You must ensure that the time on your ESXi hosts is synchronized before you inst
 
 |CPU micro-architecture
 |x86-64-v2 or higher
-|OpenShift 4.13 and later are based on RHEL 9.2 host operating system which raised the microarchitecture requirements to x86-64-v2. See the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/9.0_release_notes/index#architectures[RHEL Microarchitecture requirements documentation]. You can verify compatibility by following the procedures outlined in link:https://access.redhat.com/solutions/7052996[this KCS article].
+|{product-title} version 4.13 and later are based on RHEL 9.2 host operating system which raised the microarchitecture requirements to x86-64-v2. See the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/9.2_release_notes/index#architectures[RHEL Microarchitecture requirements documentation].
```

Comment:
```suggestion
|{product-title} version 4.13 and later are based on the {op-system-base} 9.2 host operating system, which raised the microarchitecture requirements to x86-64-v2. See the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/9.2_release_notes/index#architectures[RHEL Microarchitecture requirements documentation].
```

Reply:
For 4.16 and 4.17, RHEL 9.4 is the host operating system, I think: https://access.redhat.com/articles/6907891

Reply:
Ok! I'll reach out to the install writers and see if this is something they'd like to update :) Thanks @dfitzmau !

CC @bscott-rh @skopacz1 👀 WDYT?

Reply:
Got confirmation from Seb that this should be covered in a separate story, so I'll create the Jira ticket and assign it to Brendan Daly. We can move forward with this PR